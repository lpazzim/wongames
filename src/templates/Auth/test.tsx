import { render, screen } from '@testing-library/react'

import Auth from '.'

describe('', () => {
  it('should render all components and children', () => {
    const { container } = render(
      <Auth title="Auth Title">
        <input type="text" />
      </Auth>
    )

    expect(screen.getByRole('heading', { name: /Auth/i })).toBeInTheDocument()

    expect(container.firstChild).toMatchSnapshot()
  })
})
