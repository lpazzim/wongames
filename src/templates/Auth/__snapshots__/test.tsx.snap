// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` should render the heading 1`] = `
<main
  class=""
>
  <h1>
    Auth
  </h1>
</main>
`;
